lms/templates/sysadmin_dashboard.html

```diff
@@ -215,8 +215,4 @@ <h3>${plot['title']}</h3>
 %endif
 
   </section>
-  <div style="text-align:right; float: right"><span id="djangopid">'Django PID': ${djangopid}</span>
-  ## Translators: A version number appears after this string
-  | <span id="edxver">${_('Platform Version')}: ${edx_platform_version}</span></div>
-</div>
```

Comment:
I see one opening div and two closing /div in the removed lines. Are you sure you did not delete the second /div by accident?

Reply:
Done